zboxcore/sdk/read_counter_ctr.go

```diff
@@ -2,40 +2,43 @@ package sdk
 
 import (
 	"sync"
-
-	"github.com/0chain/gosdk/zboxcore/blockchain"
 )
 
 type blobberReadCounter struct {
 	m  map[string]int64
 	mu *sync.RWMutex
 }
 
-var brc blobberReadCounter
+var brc *blobberReadCounter
```

Comment:
Can we just go with this so no need for `InitReadCounter` func?
```suggestion
var brc = &blobberReadCounter{
  m:  make(map[string]int64),
  mu: &sync.RWMutex{},
}
```